tests/hostfs_uds/host_uds_server.c

```diff
@@ -10,9 +11,13 @@
 #define PATH_MAX 4096
 
 const char* hostdir = NULL;
+char ready_path[PATH_MAX];
+bool ready = false;
```

Comment:
You could get rid the ``ready`` variable and just do this check:

```
    if (*ready_path)
        unlink(ready_path);
```

Globals and statics a guaranteed to be zero-initialized in C.

Reply:
Fixed.